### PR TITLE
Editing text based on feedback

### PR DESCRIPTION
**Pull request by sishida**
Signed-off-by: Sarah Ishida <sishida@us.ibm.com>

Adding PR for https://github.com/eclipse/codewind/issues/150.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,28 +7,30 @@
 Create and develop cloud-native, containerized web applications from VS Code.
 
 ## Installing Codewind
+You can install Codewind locally on VS Code. For more information about installing Codewind, see [Getting started: Codewind for VS Code](https://www.eclipse.org/codewind/mdt-vsc-getting-started.html).
```
Comment (tetchel) [inline README.md]: i think "in VS Code" is better than "on" here

File: README.md [hunk 2]
```diff
@@ -7,28 +7,30 @@
...
 Prerequisites
 - Install [VS Code](https://code.visualstudio.com/download).
 - Install Docker.
 - If you use Linux, you also need to install Docker Compose.
 
 Complete the installation:
 1. Find Codewind for VS Code in the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind) or by searching for `Codewind` in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
-2. Go to the **Explorer** view group and open the **Codewind** view.
-3. Click **Install** when prompted. The download is approximately 1 GB.
+2. Open the **Codewind** view in the [Explorer view group](https://code.visualstudio.com/docs/getstarted/userinterface) or enter `Focus on Codewind View` into the [**Command Palette**](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette). If you do not see the Codewind view in either the Explorer view or the **C**ommand Palette**, the extension did not install correctly.
```
Comment (tetchel) [inline README.md]: I don't know how helpful linking to the UI page is here, given that it does not use the term "view group" or actually point you to what the explorer is. Maybe we could use a screenshot that highlights the Explorer tab and the Codewind view.

Comment (tetchel) [reply README.md]: also small typo `**C**ommand Palette**`

**Comment by sishida** [comment]
@tetchel Could you please merge this PR if everything looks good to go? (I still can't merge things myself yet.)

**Comment by tetchel** [comment]
waiting for @thisguy-1

**Pull request merged**